lib/grape-swagger/doc_methods.rb

```diff
@@ -103,8 +103,8 @@ def parse_params(params, path, method, options = {})
         values               = value.is_a?(Hash) ? value[:values] : nil
         enum_or_range_values = parse_enum_or_range_values(values)
 
-        if value.is_a?(Hash) && value.key?(:documentation) && value[:documentation].key?(:param_type)
-          param_type = value[:documentation][:param_type]
```

Comment:
If there's no `documentation` key this will fail, it's clearly a change here that is not caught in tests (and should be probably), right?